djautotask/api_rest.py

```diff
@@ -0,0 +1,359 @@
+import json
+import logging
+import re
+
+import requests
+from django.conf import settings
+from retrying import retry
+from django.core.cache import cache
+
+from djautotask.utils import DjautotaskSettings
+
+CW_RESPONSE_MAX_RECORDS = 500  # The greatest number of records Autotask
+# will send us in one response.
+RETRY_WAIT_EXPONENTIAL_MULTAPPLIER = 1000  # Initial number of milliseconds to
+# wait before retrying a request.
+RETRY_WAIT_EXPONENTIAL_MAX = 10000  # Maximum number of milliseconds to wait
+# before retrying a request.
+AT_DEFAULT_PAGE = 1  # CW Pagination is 1-indexed
```

Comment:
I don't think this is being needed or being used.

Reply:
Good point. RETRY_WAIT_EXPONENTIAL_MAX is used in fetch_resource, but AT_DEFAULT_PAGE is not. Thanks